core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -107,6 +109,18 @@
   private SqlFunctions() {
   }
 
+  /** SQL TO_BASE64(string) function*/
+  public static String toBase64(String s) {
+    if (s == null) {
+      return null;
+    }
+    try {
+      return Base64.encodeBase64String(s.getBytes("UTF-8"));
```

Comment:
Calcite is Java8. Please use https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html

Reply:
Thanks. I have changed it.